cypress/e2e/room_board/collabora_files_room_board/addEditDeleteCollaboraTextFileInRoomBoard.feature

```diff
@@ -0,0 +1,75 @@
+@regression_test
+@stable_test
+@group-V
```

Comment:
We shouldn't use so many groups anymore. So maybe you can ask Uzair, which group this test should be added later or what to do about it.